exchanges/graphcache/src/operations/query.ts

```diff
@@ -448,7 +448,7 @@ const resolveResolverResult = (
     // Check whether values of the list may be null; for resolvers we assume
     // that they can be, since it's user-provided data
     const _isListNullable =
-      !store.schema || isListNullable(store.schema, typename, fieldName);
+      store.schema ? isListNullable(store.schema, typename, fieldName) : false;
```

Comment:
Generally find it easier to read, feel free to change

Reply:
Oh, it's generally just an opinion 😆 but I avoid adding ternaries for boolean expressions since if you just add one more expression you'd remove ternariers anyway, which means that this either will discourage you from adding more or make the expression more obscure when it's kept.